.gitmodules

```diff
@@ -57,3 +57,7 @@
 [submodule "catch2/Catch2"]
 	path = catch2/Catch2
 	url = https://github.com/catchorg/Catch2.git
+
+[submodule "spi_nand_flash/dhara"]
+	path = spi_nand_flash/dhara
+	url = https://github.com/dlbeer/dhara.git
```

Comment:
Please add sbom_dhara.yml for the upstream library, as done in https://github.com/espressif/idf-extra-components/blob/master/libpng/sbom_libpng.yml